cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2712,6 +2712,15 @@
   </property>
 
   <property>
+    <name>security.authorization.cache.max.entries</name>
+    <value>10000</value>
+    <description>
+      Number of entries to hold in the container authorization cache. This setting only takes effect if
+      ${security.authorization.cache.enabled} is set to true.
```

Comment:
Why need two separate setting? Can't we just use this one and if it is 0, that means no cache.